app/controllers/ConfirmSubmissionController.scala

```diff
@@ -162,7 +161,7 @@ class ConfirmSubmissionController @Inject()(mandationStatusCheck: MandationStatu
           FailureAuditModel(user.vrn, periodKey, user.arn, error.message),
           Some(controllers.routes.ConfirmSubmissionController.submit(periodKey).url)
         )
-        logger.warn(s"[ConfirmSubmissionController][submitVatReturn] Error returned from vat-returns service: $error")
```

Comment:
should this be an error log?

Reply:
yeah i think it should be, it throws an internal server error when the user submits their answers on the confirmation page, and the user is shown an internal server error page. this is not a usual validation error on a form, so i think it should be error.